resources/feed-template.xml

```diff
@@ -3,6 +3,7 @@
 <rss version="2.0"
     xmlns:content="http://purl.org/rss/1.0/modules/content/"
     xmlns:dc="http://purl.org/dc/elements/1.1/"
+    xmlns:media="http://search.yahoo.com/mrss/"
```

Comment:
It is the same one. Check this [here](http://www.rssboard.org/media-rss#namespace-declaration)